legend-sdlc-protocol-pure/src/main/java/org/finos/legend/sdlc/protocol/pure/v1/PureToEntityConverter.java

```diff
@@ -41,5 +43,13 @@ protected String getEntityPath(PackageableElement element)
     {
         return element.getPath();
     }
+
+    ImmutableSet<String> supportedClassifiers = Sets.immutable.withAll(classToClassifier.values());
```

Comment:
Move the field up next to the declaration for classToClassifier, and make it private final. Also, use this.classToClassifier instead of just classToClassifier.